Homework/week-2/program.js

```diff
@@ -0,0 +1,7 @@
+ total = 0;
+ 
+ for(pos =2; pos <= process.argv.length-1; pos++){
```

Comment:
Don't forget to declare your variable here either, i.e. `for (var pos = 2; pos <= process.argv.length - 1; pos++)`

Also, it might be more readable to say `pos < process.argv.length`, what you have here is absolutely fine, but can be more difficult to immediately parse what your intentions are.